samples/wifi/sta/sample.yaml

```diff
@@ -58,3 +58,19 @@ tests:
       - thingy53/nrf5340/cpuapp
     platform_allow: thingy53/nrf5340/cpuapp
     tags: ci_build sysbuild ci_samples_wifi
+  sample.nrf7002eb.nrf54l15pdk.shell:
```

Comment:
New test scenarios should be designed only for `nrf54l15dk/nrf54l15/cpuapp`.

Platform `nrf54l15pdk/nrf54l15/cpuapp` will be deprecated soon.